### ISSUE TITLE
connectivity tests expected to be configable

### ISSUE THREAD

**Issue by l1b0k** [cilium/cilium-cli#222]
expected to config the address for the test `world` 

```
❌ [pod-to-world] cilium-test/client-5f479986f6-vb5qm (192.168.8.84) -> google.com (google.com)
```

**Comment by ti-mo** [comment]
@l1b0k This probably won't happen in the near future since the tests and their policies are currently tightly-coupled. I agree we should probably use another domain for testing purposes, we're open to suggestions. (I assume you're trying to run this from China)

**Comment by tklauser** [comment]
This was fixed by #1267, the address can now be configured using `cilium connectivity test --external-target example.com`.
